docs/source/Fabric-FAQ.rst

```diff
@@ -242,14 +242,12 @@ Ordering Service
 
 :Answer:
   A consensus plugin needs to implement the ``Consenter`` and ``Chain``
-  interfaces defined in the `consensus package`_. There are two plugins built
-  against these interfaces already: raft_ and kafka_. You can study them to take
-  cues for your own implementation. The ordering service code can be found under
+  interfaces defined in the `consensus package`_. There is a plugin built
+  against raft_ . You can study it to learn more for your own implementation. The ordering service code can be found under
   the `orderer package`_.
 
 .. _consensus package: https://github.com/hyperledger/fabric/blob/master/orderer/consensus/consensus.go
 .. _raft: https://github.com/hyperledger/fabric/tree/master/orderer/consensus/etcdraft
-.. _kafka: https://github.com/hyperledger/fabric/tree/master/orderer/consensus/kafka
 .. _orderer package: https://github.com/hyperledger/fabric/tree/master/orderer
```

Comment:
Shouldn't these be pointing to release-2.0.0? Master can change at any time so I'd expect references in doc to point to the more stable release branches.

Reply:
Yes. Thanks Matt. I've updated those now. I need to talk to Dave about links like this across the rest of our docs.

Reply:
Brett thinks he has a way to dynamically detect the branch so we never have to update these when a branch is cut. Will try to add that tomorrow.